logic/account.js

```diff
@@ -496,13 +496,11 @@ Account.prototype.verifyPublicKey = function (publicKey) {
 			throw 'Invalid public key, must be a string';
 		}
 		// Check length
-		if (publicKey.length < 64) {
+		if (publicKey.length !== 64) {
 			throw 'Invalid public key, must be 64 characters long';
 		}
-		// Check format
-		try {
-			Buffer.from(publicKey, 'hex');
-		} catch (e) {
+
+		if (!/^[a-fA-F0-9]+$/.test(publicKey)) {
```

Comment:
can we keep these checks in one place across the project? Same logic is used in z_schema file